### PR TITLE
[Validator] Add `WordCount` constraint French translation

### PR DESCRIPTION
**Pull request by alexandre-daubois**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

:information_source: Requires #57716 to be merged first

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 1]
```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target>Cette URL doit contenir un domaine de premier niveau.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Cette valeur est trop courte. Elle doit contenir au moins {{ min }} mots.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>This value is too long. It should contain {{ max }} words or less.</source>
+                <target>Cette valeur est trop longue. Elle doit contenir {{ max }} mots ou moins.</target>
```
Comment (smnandre) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]:
```suggestion
                <target>Cette valeur est trop longue. Elle doit contenir au maxiumum {{ max }} mots.</target>
```

wdyt ?

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 2]
```diff
@@ -442,6 +442,14 @@
                 <source>This URL is missing a top-level domain.</source>
                 <target>Cette URL doit contenir un domaine de premier niveau.</target>
             </trans-unit>
+            <trans-unit id="114">
+                <source>This value is too short. It should contain at least {{ min }} words.</source>
+                <target>Cette valeur est trop courte. Elle doit contenir au moins {{ min }} mots.</target>
```
Comment (94noni) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Perhaps picky here, just passing by

What if {{ min }} is 1 ?

How others core translations handle singular plural if any?

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 1 above)
Comment (alexandre-daubois) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Suits me fine :+1:

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 2 above)
Comment (alexandre-daubois) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: That's right, I updated this PR and the other one to pluralize the message

**Comment by xabbuh** [comment]
we should probably merge #57752 first

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alexandre-daubois.

**Pull request merged**